### PR TITLE
Change tidb_memory_usage_alarm_ratio scope to instance 

### PR DESCRIPTION
**Pull request by wshwsh12**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by crazycs520 — APPROVED** [review]
LGTM

**Comment by XuHuaiyu** [comment]
Is there a related PR in TiDB?

File: system-variables.md [hunk 1]
```diff
@@ -922,8 +922,9 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 
 ### `tidb_memory_usage_alarm_ratio`
 
-- 作用域：SESSION
-- 默认值：`0.8`
+- 作用域：`INSTANCE`
+- 默认值：0.8
+
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
```
Comment (TomShawn) [inline system-variables.md:928]:
```suggestion
- 默认值：`0.8`
- TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
```

**Comment by wshwsh12** [comment]
> related PR in TiDB?

@XuHuaiyu  We can see the variable is in process global variables list.  
https://github.com/pingcap/tidb/blob/7259784c683c44841ffb4cd202ed96ac0d4738ca/sessionctx/variable/tidb_vars.go#L744
I did not modify the logic of tidb, but just corrected the document.

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**